project_report/__init__.py

```diff
@@ -121,6 +143,14 @@ def project_size_in_terabytes(self):
         project_size = self.get_folder_size(self.project_delivery)
         return (project_size/1000000000000.0)
 
+    def parse_date(self, date):
+        if not date:
+            return 'None'
+        d = parser.parse(date)
+        datelist = [d.year, d.month, d.day]
+        return '-'.join([str(i) for i in datelist])
```

Comment:
dateutil is not part of the standard library and is not currently maintained. See below for an implementation with datetime:
```python
>>> date = '2017-11-07T16:18:06.673000'
>>> d = datetime.strptime(date, '%Y-%m-%dT%H:%M:%S.%f')
>>> d.strftime('%Y-%m-%d')
'2017-11-07'
```